### PR TITLE
Update announcement red dot

### PR DESCRIPTION
**Pull request by mkitt**
The red dot now goes away after a user has seen the announcement (and has no unread notifications) rather than waiting for it to be dismissed. 

Tried to leave things in a nicer place than I found them.

 🏓 🔴 

[#139878137](https://www.pivotaltracker.com/story/show/139878137)

### CODE REVIEW

**Comment by mkitt** [comment]
@rynbyjn happy to walk you through this one too.

File: src/containers/notifications/Notifications.js [hunk 1]
```diff
@@ -172,7 +184,7 @@ class Notifications extends Component {
             /> :
             null
         }
-        { hasAnnouncementNotification &&
+        { announcementIsEmpty &&
```
Comment (rynbyjn) [inline src/containers/notifications/Notifications.js]: It seems to me that this should probably be `!announcementIsEmpty`..

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Big PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">Nice, more deletions than insertions :red_circle:</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/containers/notifications/NotificationsContainer.js [hunk 2]
```diff
@@ -217,7 +228,7 @@ class NotificationsContainer extends Component {
               /> :
               null
           }
-          { hasAnnouncementNotification &&
+          { announcementIsEmpty &&
```
Comment (rynbyjn) [inline src/containers/notifications/NotificationsContainer.js]: Same here with the `!`..

**Pull request merged**